src/child/containers/sidebars/favourites/Favourites.js

```diff
@@ -109,15 +109,7 @@ class Favourites extends Component {
 
     toggleFavourite(stockCode) {
         this.props.dispatch(toggleFavourite(stockCode));
```

Comment:
Does it make sense to also pull this up to `selectNewFavourite` in the Sidebar container? (If so, I'd probably also rename `selectNewFavourite`, maybe to `toggleFavourite`?)